README.md

```diff
@@ -778,7 +778,7 @@ algorithms, knowledgebase and AI technology.
 * [Clip Blast](http://www.clipblast.com)
 * [DailyMotion](http://www.dailymotion.com)
 * [Deturl](http://deturl.com) - Download a YouTube video from any web page.
-* [DownloadHealper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
+* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```

Comment:
- - - - - - - - - - - - - - - - - - @@@@@@@@@[[[[_**
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```

```

```

```

```

```
**_]()]()](
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```suggestion
* [DownloadHelper](http://www.downloadhelper.net) - Download any video from any websites, it just works!
```

```
)]()